### PR TITLE
Feature: Market Details for Scalar Market

### PR DESCRIPTION
**Pull request by andre-meyer**

### CODE REVIEW

File: app/src/MarketTable_WithScalar/index.js [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
-import { oneDecimal } from "utils/constants";
+//import { oneDecimal } from "utils/constants";
 import { calcSelectedMarketProbabilitiesFromPositionProbabilities } from "utils/probabilities";
 
 const { BN } = Web3.utils;
 
 const cx = cn.bind(style);
 
+/* eslint-disable */
+const markdownRenderers = {
```
Comment (denisgranha) [inline app/src/MarketTable_WithScalar/index.js]: move this to a general package? something like utils

File: app/src/MarketTable_WithScalar/index.js [hunk 2]
```diff
@@ -54,6 +69,7 @@ const MarketTable = ({
     );
 
     if (stagedTradeAmounts != null) {
+      /*
```
Comment (denisgranha) [inline app/src/MarketTable_WithScalar/index.js]: why is this commented?

File: app/src/conf/config.rinkeby.json [hunk 3]
```diff
@@ -3,6 +3,8 @@
   "markets": [
     {
       "questionId": "0xda66c02003000000000000000000000000000000000000000000000000000000",
+      "dataSource": "DataSource",
+      "dataSourceUrl": "https://example.org",
```
Comment (denisgranha) [inline app/src/conf/config.rinkeby.json]: I think it should be taking it from the API right? so this is no longer needed

Comment (andre-meyer) [reply app/src/conf/config.rinkeby.json]: Not for now, as we're using the config again for a moment to hardcode the scalar properties.

**Pull request merged**